content/sensu-core/1.4/guides/monitor-the-monitor.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Monitoring Sensu with Sensu"
+description: "Strategies and best practices for monitoring Sensu with Sensu"
+version: "1.4"
+weight: 10
+previous: ../securing-redis
+menu:
+  sensu-core-1.4:
+    parent: guides
+---
+
+In this guide, we'll walk you through the best practices and strategies for monitoring Sensu with Sensu. By the end of the guide, you should have a thorough understanding of what is required to ensure your Sensu components are properly monitored, including:
+
+* How to monitor your Sensu server and Sensu API instance(s)
+* How to monitor your Uchiwa Dashboard instance(s)
+* How to monitor your RabbitMQ instance(s)
+* How to monitor your Redis instance(s)
+
+_NOTE: In order to completely monitor a Sensu stack (Sensu services, Redis, RabbitMQ), you will need to have at least one other independent Sensu stack to do so. This is due to a single Sensu stack can not monitor itself completely, as if some components are down, Sensu will not be able to alert on itself properly_
+
+_NOTE: This guide assumes you are not using Sensu clustering, RabbitMQ clustering or using Redis Sentinels. You can still monitor each server using the below described strategies, but it may not be the best way to do so._
+
+## Objectives
+
+We'll cover the following in this guide:
+
+* [Monitoring Sensu](#monitoring-sensu)
+  * [Monitoring Sensu Server](#monitoring-sensu-server)
+  * [Monitoring Sensu API](#monitoring-sensu-api)
+  * [Monitoring Uchiwa](#monitoring-uchiwa-dashboard)
+* [Monitoring RabbitMQ](#monitoring-rabbitmq)
+* [Monitoring Redis](#monitoring-redis)
```

Comment:
Your first paragraph is a great statement of the guide's objectives. I'd suggest removing this Objectives section and making the items in the first list into links. That way you get to the content sooner.